examples/switzerland/PlaceInformation_001_Stop_PlaceRef_Request.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.siri.org.uk/siri" xmlns:ojp="http://www.vdv.de/ojp" version="1.1-dev" xsi:schemaLocation="http://www.siri.org.uk/siri ../../OJP.xsd">
```

Comment:
The definition `xsi:schemaLocation="http://www.siri.org.uk/siri ../../OJP.xsd"` is kind of a hack and up to now my biggest concern with the pull request.
SIRI is defined by `http://www.siri.org.uk/siri` and usually does not know anything about OJP. Only by referencing `../../OJP.xsd` here the root element definition `<OJP>` and the other elements from `OJP.xsd` are defined.

So OJP is based on SIRI but my XML/XSD knowledge is not good enough yet to propose a valid solution: Do we need an own namespace for this SIRI enhancement of `OJP.xsd` (apart from `xmlns:ojp="http://www.vdv.de/ojp"` which contains our own definitions not based on SIRI)?

Defining a `schemaLocation` is optional, also see https://stackoverflow.com/questions/4998063/one-xml-namespace-equals-one-and-only-one-schema-file:

> Some XML processors tolerate multiple <import> for the same namespace, and essentially amalgamate all of the schemaLocation into a single target.
> 
> Other processors are stricter, and decide that only one <import> per target namespace is valid. I think this is more correct, when you consider that schemaLocation is optional.
> 
> In addition to the VS and xmllint examples you gave, Xerces-J is also super-strict, and ignores subsequent <import> for the same target namespace, giving much the same error as xmllint does. XML Spy, on the other hand, is much more permissive (but then, XML Spy's validation is notoriously flaky)
> 
> To be safe, you should not have these multiple imports. A given namespace should have a single "master" document, which in turn has an <include> for each sub-document. This master is often highly artificial, acting only as a container. for these sub-documents.